src/app/SharpRaven/Utilities/Serializer.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Text;
+
+namespace SharpRaven.Utilities
+{
+    public static class Serializer
+    {
+        public static byte[] SerialiseBytes(this object obj)
```

Comment:
@rolandh: No, but as `JsonPacket.ToString()` already does serialisation, it should be easy enough to add something like that. Not sure it belongs in the `JsonPacket` class, but it's not very important, I think.